specification/purview/data-plane/Azure.Analytics.Purview.DataPolicies/preview/2022-02-01-preview/purviewDataPolicy.json

```
@@ -0,0 +1,822 @@
+          "type": "string"
+        },
+        "description": {
+          "description": "The description of role",
+          "type": "string"
+        },
+        "cnfCondition": {
+          "$ref": "#/definitions/CnfCondition"
+        },
+        "dnfCondition": {
+          "$ref": "#/definitions/DnfCondition"
+        },
+        "version": {
+          "format": "int64",
+          "description": "The version of role",
+          "type": "integer"
+        }
+      }
+    },
+    "DataPolicyRole": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "The Id of role",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of role",
+          "type": "string"
+        },
+        "type": {
+          "description": "The type of role",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/DataPolicyRoleProperties"
+        }
+      }
+    },
+    "DataPolicyRoleList": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataPolicyRole"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "DataPolicyBinding": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "The type of reference",
+          "default": "DataSourceReference",
+          "type": "string"
+        },
+        "referenceName": {
+          "description": "The name of data source",
+          "type": "string"
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData"
+        }
+      }
+    },
+    "DataPolicyBindingList": {
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataPolicyBinding"
+          }
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "api-version": {
+      "in": "query",
+      "name": "api-version",
+      "description": "The api version to use.",
+      "required": true,
+      "type": "string",
+      "x-ms-client-name": "ApiVersion"
+    },
+    "Endpoint": {
+      "in": "path",
+      "name": "Endpoint",
+      "description": "The endpoint of your Purview account. Example: https://{accountName}.purview.azure.com.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "client",
+      "x-ms-skip-url-encoding": true
+    }
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
```

Please specify the actual scope name in the oauth2 securityDefinition both for SDK generation and the public REST API docs.
```suggestion
        "https://purview.azure.net/.default": "impersonate your user account"
```

---

Thanks, I have updated to "https://purview.azure.net/.default".

One question, should there be `.default` at the end of the scope?